app/views/articles/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<h1>New article</h1>
+
+<div class="flex flex-wrap -mx-3 mb-6">
+  <%= form_with(model: @article) do |f| %>
+    <% if @article.errors.any? %>
+      <div style="color: red">
+        <h2><%= pluralize(@article.errors.count, "error") %> prohibited this article from being saved:</h2>
+
+        <ul>
+          <% @article.errors.each do |error| %>
+            <li><%= error.full_message %></li>
+          <% end %>
+        </ul>
+      </div>
+    <% end %>
+
+    <div>
+      <%= f.label :title, style: "display: block" %>
+      <%= f.text_field :title %>
+    </div>
+    <div>
```

Comment:
It would be great to add a new line between these two lines. It will make the code much more readable. We have to remember that the code isn't only for us now. It's also for our future selves. We should be kind to them.

(ditto throughout)